net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBGPNeighbor.xml

```diff
@@ -32,4 +32,16 @@
     <label>Send Defaultroute</label>
     <type>checkbox</type>
   </field>
+  <field>
+    <id>neighbor.linkedRoutemapIn</id>
+    <label>Route-Map In</label>
+    <type>dropdown</type>
+    <help>Route-Map for Inbound direction</help>
```

Comment:
Done. I would write it with the dash and treat it like a "name" ... like "Update-Source".